<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 64, 77)
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_win_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{background-color: rgb(61, 64, 77)}

QPushButton{
	padding-top: 5px;
	background-color: rgb(84, 89,105); 
	color:rgb(209,210,214);
	border:none; 
	border-top-right-radius:5px;
	border-top-left-radius:5px;
}
QPushButton#btn_menu{
	background-color: rgb(73, 102,153); 
}

QPushButton#btn_min, QPushButton#btn_max{
	background-color:rgba(81,83,95,0);
	padding-top: 0px;
	border:none;
	border-top-right-radius:0px;
	border-top-left-radius:0px;
}
QPushButton#btn_min:hover, QPushButton#btn_max:hover{background-color:rgb(81,83,95)}

QPushButton#btn_exit{
	padding-top: 0px;
	border:none;
	border-top-right-radius:0px;
	border-top-left-radius:0px;
	background-color: rgba(73, 102,153,0);
}
QPushButton#btn_exit:hover{background-color: rgb(240, 96,77);}
QPushButton#btn_new{background-color: rgba(73, 102,153,0);}
QPushButton:pressed{
	background-color: rgb(232,232,232); 
	color:rgb(95,145,230)
}
QPushButton#btn_menu:checked{
	background-color: rgb(245, 245,245); 
	color:rgb(95,145,230)
}

QTabBar{
    border-top: 30px solid #3d404d;
    background-color: rgb(245,245,245);
    qproperty-drawBase:0;
    }

QTabBar::tab {
    border: 1px solid #3d404d;
    background: rgb(84,89,105);
	color:rgb(209,210,214);
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    border-bottom : none;
    padding-left: 5px;
    padding-right: 5px;
}
QTabBar::tab:tear {
        border-bottom : none;
        padding: 7 20px;
}
QTabBar::tab:selected:active {
    background: rgb(232,232,232);
    color:rgb(95,145,230)
}
QTabBar::tab:hover{background: rgb(107,114,132);}
QTabBar::close-button:hover {background: rgb(107,114,132);}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layotu_btn_tabs">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> FullAxis </string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_new">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string> + Nueva Prueba </string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_min">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Rashi Hebrew Thin</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>—</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_max">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Rashi Hebrew Thin</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>□</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_exit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Rashi Hebrew Thin</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_menu">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>58</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>58</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245,245,245); 
</string>
      </property>
      <layout class="QHBoxLayout" name="layout_menu">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245,245,245); 
</string>
      </property>
      <layout class="QHBoxLayout" name="layout_central">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
